src/components/BrowserCell/BrowserCell.react.js

```diff
@@ -304,6 +305,30 @@ export default class BrowserCell extends Component {
     return contextMenuOptions;
   }
 
+  async executeSript(script) {
+      try {
+        const object = Parse.Object.extend(
+          this.props.className
+        ).createWithoutData(this.props.objectId);
+        const response = await Parse.Cloud.run(
+          script.cloudCodeFunction,
+          { object: object.toPointer() },
+          { useMasterKey: true }
+        );
```

Comment:
The indentation here really varies from line to line, the liner doesn't correct this; @dblythy do we use prettier for this in Parse Server?

Reply:
shall i format it as how it was previously ?

Reply:
formatted it as how it was before